src/visitor/RewriteAllSubExprVisitor.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef VISITOR_REWRITEALLSUBEXPRVISITOR_H_
+#define VISITOR_REWRITEALLSUBEXPRVISITOR_H_
+
+#include "common/expression/Expression.h"
+#include "visitor/ExprVisitorImpl.h"
+
+namespace nebula {
+namespace graph {
+
+class RewriteAllSubExprVisitor final : public ExprVisitorImpl {
```

Comment:
Plz add a comment to explain the visitor's function.

Reply:
Which function?

Reply:
Emm.Purpose of the visitor.